src/styles/components/_Buttons.scss

```diff
@@ -0,0 +1,116 @@
+.Btn {
+	@include font-heading;
+	@include text-std;
+	border: 1px solid $hm-primary;
```

Comment:
Border is the same as the background on all styles, perhaps we should set `border: 0;` and remove the colour declarations below?